src/components/Popup.js

```diff
@@ -30,18 +31,20 @@ export default class Popup {
 
   setEventListeners() {
     // handle close by close button click
-    const closeButton = this._popupElement.querySelector(
-      ".modal__close-button"
-    );
+    const closeButton = this.popupElement.querySelector(".modal__close-button");
     closeButton.addEventListener("click", () => {
       this.close();
     });
 
     // handle close by overlay click
-    this._popupElement.addEventListener("click", (evt) => {
+    this.popupElement.addEventListener("click", (evt) => {
       if (evt.target.classList.contains("modal")) {
         this.close();
       }
     });
   }
+
+  resetForm() {
+    this._form.reset();
```

Comment:
`this._form = this.popupElement.querySelector("form");` is not needed in `Popup` because the image modal doesn't have any form

And `resetForm` should be in `PopupWithForm` because of that